pom.xml

```diff
@@ -425,6 +425,24 @@
               </resources>
             </configuration>
           </execution>
+          <execution>
+            <id>copy-google-assistant-images</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${basedir}/addons/ios/google-assistant</outputDirectory>
```

Comment:
@Confectrician I'm not sure this is right, as I understand it the goal should be to have the images under `_addons_ios/ext/google-assistant/doc` (because that's where the prepare-docs scripts expects them to be).
Same as for instance https://github.com/openhab/openhab-docs/tree/final/_addons_ios/openhabcloud

So I would move this execution step below the one copying USAGE.md to `_addons_ios/ext/google-assistant/readme.md` (to be sure the folder structure exists).

Reply:
See https://github.com/openhab/openhab-docs/pull/721#issuecomment-405249391
Should be tested and working after we had a longer discussion about it.

I can test it again for sure, i f you want.

openhab cloud had no problems so far and is included for ages if i remember correctly.

Reply:
Pls keep in mind that the prepare docs groovy script does some work to afterwards.
(So this could be done on another place already.)

I didn't look behind the complete process at all and just copied stuff and also trusted @ThomDietrich after his investigation of the problem.^^

The copying process is something we should simplify or at least make more read and understandable in the future anyway.